src/components/ABDM/LinkAbhaNumber/CreateWithAadhaar.tsx

```
@@ -682,18 +682,22 @@ function ChooseAbhaAddress({
         {validateRule(
           healthId.length >= 4,
           t("abha_address_validation_length_error"),
+          false,
         )}
         {validateRule(
           isNaN(Number(healthId[0])) && healthId[0] !== ".",
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)